assets/css/styles.scss

```diff
@@ -1,5 +1,6 @@
 @charset 'utf-8';
 
+@import 'leaflet';
```

Comment:
Let's remove the `assets/css/_leaflet.scss` file from here, and this import line. The reason why it's not a good idea to copy a library asset is because it makes it harder to remember to copy the file over every time you want to upgrade the library.

Instead, let's try to find a way to import the CSS from `node_modules`.